### ISSUE TITLE
How to use the train and val monitors?

### ISSUE THREAD

**Issue by MrRobot2211** [stanfordmlgroup/ngboost#49]
Hi, I was wondering which is the purpose of the arguments train_loss_monitor and val_loss_monitor. Sklearn models usually include a monitor argument that allows  for early stopping. Is that the idea?

**Comment by tonyduan** [comment]
Yes, the idea is that this allows for a custom metric to monitor for purposes of early stopping. Otherwise the `fit` method will default to the specified scoring rule at instantiation.
